src/missileWeaponData.cpp

```diff
@@ -37,4 +37,30 @@ string getMissileSizeString(EMissileSizes size)
     }
 }
 
+const float MissileWeaponData::convertSizeToCategoryModifier(EMissileSizes size)
+{
+    switch(size)
+    {
+        case MS_Small:
+            return 0.5;
+        case MS_Medium:
+            return 1.0;
+        case MS_Large:
+            return 2.0;
+        default:
+            return 1.0;
+    }
+}
+
+const EMissileSizes MissileWeaponData::convertCategoryModifierToSize(float size)
+{
+    if(0.5 == size)
```

Comment:
Might want to do something like `if (std::abs(size - 0.5) < 0.1)` to allow for floating point inaccuracies.

Reply:
Makes sense even if 0.1 might be a little overkill :) Don't see it upgrading to 0.1 increments though...